packages/nodes/mention/src/withMention.ts

```diff
@@ -28,6 +28,15 @@ export const withMention: WithOverride<{}, MentionPlugin> = (
     deleteBackward(unit);
   };
 
+  editor.insertBreak = () => {
+    const currentMentionInput = findMentionInput(editor);
+    if (currentMentionInput && Node.string(currentMentionInput[0]) === '') {
```

Comment:
I don't understand the empty input special case. Also, i believe the action on insertBreak should be a no-op, it should not remove the mention. We have escape to remove the mention, enter should do nothing, regardless of whether it's empty or not.